app/views/candidate_interface/submitted_application_form/start_apply_again.html.erb

```diff
@@ -16,10 +16,12 @@
     </p>
     <p class="govuk-body">You can only apply to one course at a time at this stage of your application.</p>
 
-    <%= button_to 'Start now', candidate_interface_apply_again_path, class: 'govuk-button govuk-button govuk-!-margin-bottom-5' %>
```

Comment:
can we keep the class `govuk-!-margin-bottom-5`?

Reply:
I looked at this and it only adds (or was it removes) an additional 5px, so I'd rather defer to the standard margin.